src/storage/src/render/sources.rs

```diff
@@ -197,13 +197,14 @@ pub fn render_source<'g, G: Scope<Timestamp = ()>>(
     needed_tokens.push(source_token);
 
     let mut outputs = vec![];
+    let mut health_output = health;
```

Comment:
just use `let (..., mut health_output, ...)` above instead of re-assigning it here!